client/me/purchases/vat-info/index.tsx

```diff
@@ -134,10 +137,12 @@ export default function VatInfoPage() {
 									ul: <ul />,
 									li: <li />,
 									contactSupportLink: (
-										<WordPressButton
-											className="vat-info__open-help-center-support"
+										// eslint-disable-next-line jsx-a11y/anchor-is-valid
+										<a
+											target="_blank"
+											href="#"
+											rel="noreferrer"
```

Comment:
I think we can solve this by making this semantically a proper link to `/help`. This way we'll have all the benefits together. No need to violate a11y rules, really:

```suggestion
										<a
											href="/help"
```